appclient/src/main/resources/schema/jboss-client_9_0.xsd

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ JBoss, Home of Professional Open Source
+~ Copyright 2019, Red Hat, Inc., and individual contributors as indicated
+~ by the @authors tag.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+<xsd:schema xmlns="http://www.w3.org/2001/XMLSchema"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            xmlns:jboss="urn:jboss:jakartaee:1.0"
+            targetNamespace="urn:jboss:jakartaee:1.0"
+            version="9.0"
+            elementFormDefault="qualified"
+        >
+
+   <xsd:annotation>
+      <xsd:documentation>
+         <![CDATA[
+
+	This is the XML Schema for the JBoss 8.0 application client deployment descriptor.
+	The deployment descriptor must be named "META-INF/jboss-client.xml" in
+	the application client's jar file.  All the application client deployment descriptors must indicate
+	the JBoss schema by using the Jakarta EE namespace:
+
+	urn:jboss:jakartaee:1.0
+
+	and by indicating the version of the schema by
+	using the version attribute as shown below:
+
+	    <jboss-client xmlns="urn:jboss:jakartaee:1.0"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="urn:jboss:jakartaee:1.0
+                            https://www.jboss.org/j2ee/schema/jboss-client_9_0.xsd"
+             version="9.0">
+	       ...
+            </jboss-client>
+
+	The instance documents may indicate the published version of
+	the schema using the xsi:schemaLocation attribute for the
+	Java EE namespace with the following location:
+
+	https://www.jboss.org/schema/jbossas/jboss-client_9_0.xsd
+
+	]]>
+      </xsd:documentation>
+   </xsd:annotation>
+
+   <xsd:import namespace="https://jakarta.ee/xml/ns/jakartaee" schemaLocation="https://jakarta.ee/xml/ns/jakartaee/application-client_10.xsd"/>
```

Comment:
Why is it ok to have JBoss schemas versioned as 9 while they builds up on Jakarta EE10?

Reply:
Because they don't relate